### PR TITLE
Moved new material from README.md into hosted docs, and add links to …

### PR DESCRIPTION
**Pull request by mseeger**
…scheduler table

*Issue #, if available:*

*Description of changes:*
Last thing needed for the release

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by mseeger** [comment]
I'll add the reshaping of Mina's addition to the basics tutorial to this PR.

**Comment by mseeger** [comment]
OK, this is ready to go

File: docs/source/tutorials/basics/concepts.rst [hunk 1]
```diff
@@ -1,109 +1,173 @@
...
+`this example <../../examples.html#launch-hpo-experiment-with-simulator-backend>`__.
+
+The main advantage of the simulator backend is that it allows for realistic
+experimentation at very low cost, and running order of magnitude faster than
+real time. A drawback is the upfront cost of generating a tabulated benchmark
+of sufficient complexity to match the real problem of interest.
+
+
+Importantly, Syne Tune is agnostic to which execution backend is being used. You
+can switch between backends effortlessly, just by changing the ``trial_backend``
```
Comment (wesk) [inline docs/source/tutorials/basics/concepts.rst:152]:
```suggestion
can easily switch between backends by changing the ``trial_backend``
```

**Review by wesk — DISMISSED** [review]
One non-blocking nit comment. Looks good

**Review by wesk — APPROVED** [review]

**Pull request merged**